example/linux/systemd/Talawa-api.sh

```
@@ -0,0 +1,52 @@
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+nvm use 20.18.0
+
+# Define the path to the tsx executable dynamically
+TSX_PATH=$(which tsx)
+
+# Define paths for development and production
+DEV_PATH="src/index.ts"
+PROD_PATH="dist/index.js"
+
+# Change to the project directory .give your path
+cd /home/purnendu/Development/talawa-api
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)